packages/shared/flows.yaml

```diff
@@ -404,6 +404,15 @@ integrations:
         output: GithubRepoFile
         endpoint: /github/files
         scopes: repo
+      github-issues-demo:
+        runs: every 5 minutes
+        auto_start: false
+        sync_type: full
+        output: GithubIssueDemo
+        scopes: public_repo
+        description: |
+          Fetches GitHub issues from our showcase repository.
+        endpoint: /github/demo-issues
```

Comment:
I know GET is the default method but I would recommend we always specify it explicitely.

Reply:
fixed